sql/src/main/java/io/crate/operation/reference/sys/cluster/ClusterSettingsExpression.java

```diff
@@ -139,11 +90,8 @@ public ClusterSettingsExpression(Settings settings, ClusterService clusterServic
         this.clusterSettings = clusterService.getClusterSettings();
         this.clusterService = clusterService;
         setDefaultValues(CrateSettings.SETTINGS);
-
-        //ApplySettings applySettings = new ApplySettings(settings, values);
-        //nodeSettingsService.addListener(applySettings);
         applyInitialSettingsAndRegisterUpdateConsumer(CrateSettings.CRATE_SETTINGS, settings);
-
+        applyInitialSettingsAndRegisterUpdateConsumer(CrateSettings.SETTINGS, settings);
```

Comment:
hm as SETTINGS includes CRATE_SETTINGS, we could remove line above. but the intention here was only to call this for the additional crate settings only. why was this required? can't we solve this differently?